README.md

```diff
@@ -115,9 +116,9 @@ make docs
 
 Benchmark results are hosted [here](https://lambdaclass.github.io/lambdaworks/bench).
 
-To run them locally, you will need `cargo-criterion`. Install it with:
+To run them locally, you will need `cargo-criterion` and `cargo-flamgraph`. Install it with:
```

Comment:
I think there's a typo, it should be `cargo-flamegraph`

Reply:
Fixed! thnaks :)